### ISSUE TITLE
Callable class/method array reference broken in 2020.01.16.001

### ISSUE THREAD

**Issue by brego** [klesun/deep-assoc-completion#144]
Hello! 

It would appear that you went a bit too far in your fix for issue https://github.com/klesun/deep-assoc-completion/issues/138.

Given

```php
// Foo.php
class Foo {
    public function bar() {}
}
```
And
```php
// other_file.php
$reference = [Foo::class, 'bar'];
```

Prior to `2020.01.16.001`, which solves https://github.com/klesun/deep-assoc-completion/issues/138, you could navigate from `[Foo::class, 'bar']` to method definition, and in the opposite direction, see all of the references from the definition.

This is useful when referencing methods without calling them. Maybe you could re-enable previous behavior as an option?

Thank you.

**Comment by klesun** [comment]
Oh, sorry, that was not intended. Thanks for vigilance =-D. Will fix asap.

**Comment by klesun** [comment]
Fixed in [2020.01.28.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/76721)
![image](https://user-images.githubusercontent.com/5202330/73271864-0decd900-41ea-11ea-95a8-68ae9b908eea.png)

**Comment by brego** [comment]
Thank you very much for your quick response @klesun!

You fixed the resolution of instanciated references, but not _non-instanciated_. In the following code, there's no instance for `Asd`, but the array is callable.

```php
$reference = [Asd::class, "ololo"]; // <-- ::class
call_user_func($reference);

```

**Comment by klesun** [comment]
Oh, sorry, you're right, my bad

**Comment by klesun** [comment]
Corrected static method references in [2020.01.28.002](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/76727)
![image](https://user-images.githubusercontent.com/5202330/73283495-d8e98200-41fb-11ea-828a-c6afadc53748.png)
